test/features/schema_metadata.feature

```diff
@@ -0,0 +1,35 @@
+@schema_metadata
+Feature: SchemaMetadata
+
+
```

Comment:
yellow: 
Background: I am logged into as an Admin persona
Given "Admin" as the persona
When I log in
Then I should see the logged in view (if this exists).

then collapse given admin and login as its the same for all to reduce duplicate text etc

would help